keyboards/dtd/ldmlKeyboard.dtd

```diff
@@ -149,7 +149,7 @@ Please see CLDR-15034 for the latest information. -->
 <!ELEMENT layer ( row+, special* ) >
 <!ATTLIST layer id NMTOKEN #IMPLIED >
     <!--@MATCH:any-->
-<!ATTLIST layer modifier NMTOKEN #IMPLIED >
+<!ATTLIST layer modifier NMTOKENS #IMPLIED >
     <!--@MATCH:any-->
```

Comment:
We should be able to restrict the set of possible modifiers here now?

Reply:
Yea with a `@MATCH` though is it a closed set ?

Reply:
We could list the allowable combos, e.g. Right + Left modifiers together is not sensible; Alt + Option not helpful (query whether Option should be included since it is roughly equivalent to Alt anyway). There's only really a smallish set of combos possible. This would also help us to keep the order of modifiers consistent.